app/src/main/java/org/liberty/android/fantastischmemo/ui/RecentListFragment.java

```diff
@@ -352,6 +401,31 @@ public synchronized void setItems(List<RecentItem> items) {
             this.items.addAll(items);
             this.notifyDataSetChanged();
         }
+
+        private void updateActionMode() {
+            Intent intent = new Intent("recent-list-update-action-mode");
+            intent.putExtra("selectedItemCount", getSelectedItemCount());
+            LocalBroadcastManager.getInstance(context).sendBroadcast(intent);
+        }
+
+        public void stopActionMode() {
```

Comment:
Will you be able to put methods that send broadcasts into your RecentListActionModeUtil?